locales/zh_CN/LC_MESSAGES/duckduckgo.po

```diff
@@ -4444,11 +4436,9 @@ msgid ""
 msgstr ""
 "您现在可以借助本工具来掌控您的网络活动痕迹，并且您的隐私将不会再泄漏出去！"
 
-#  Marked fuzzy because of translated trademark
-#, fuzzy
 msgctxt "SERP footer content"
 msgid "You're in control. Customize the look-and-feel of DuckDuckGo."
-msgstr "自由定制网站外观。"
+msgstr "自由定制 DuckDuckGo 外观。"
```

Comment:
Hello @b9a1. Thank you very much for these translation improvements.
I was just wondering, for this sentence do you think 「自由定制 DuckDuckGo 的外观。」 would be better? Or is it OK as is?

Reply:
Both sound fine to me, since 的 can be often omitted. But I just noticed there would be a line break between 外观 without it, so it’s indeed better included to prevent splitting the word. Thank you for bringing this up.

Reply:
Ah, good thinking - I hadn't considered that. Sounds good and thanks again! 👍